### PR TITLE
[6.x] DS-855 Prep Bolt in Drupal

### PR DESCRIPTION
**Pull request by danielamorse**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-855

## Summary

Prepare Bolt to be moved into our Drupal base theme.

## Details

- Setup React as external dependency, i.e. it will be on the window not bundled with Webpack
  - Required adding CDN link to React in head of PL demos, test environments, and microjourneys editor iframe
- Fix mismatched dependency warnings
  - Required upgrading `react-html-parser`
- Add `async` option to `bolt` cli, required by Drupal
- Move drupal-specific configuration directly into Bolt
- Enable `watch` command to work in Drupal environment
- Lock `scroll-js` to v3.1.1 to prevent Node version error

## How to test

- Review code
- Verify Bolt commands work as expected (start, build, etc)

To test this PR in Drupal you must have Saleshub setup locally. For now just review this code in Bolt and verify there are no regressions to Bolt. Any additional changes required for Drupal will be handled in future PRs.

## Release notes

**Breaking Change**
Microjourneys editor will require React to be passed in via the `scripts` prop. See [example in PR](https://github.com/boltdesignsystem/bolt/pull/2578/files#diff-c7bb7897e243c1a197a0829c37d1ddbed0fd71db50bf2ac2e4999e34ff7e6cd1R4). This loads React in the grapes JS editor iframe. It is required because bolt-global.js is loaded in the iframe and bolt-global.js expects React to be on the window.

### CODE REVIEW

File: packages/website-ui/uikit-workshop/package.json [hunk 1]
```diff
@@ -81,14 +81,14 @@
...
-    "react-html-parser": "^2.0.2",
+    "html-react-parser": "^3.0.4",
     "react-popper-tooltip": "^2.10.1",
     "redux": "4.0.5",
     "redux-thunk": "^2.3.0",
     "reselect": "^4.0.0",
     "sass-loader": "^8.0.2",
     "scriptjs": "^2.5.9",
-    "scroll-js": "^3.1.1",
+    "scroll-js": "3.1.1",
```
Comment (colbytcook) [inline packages/website-ui/uikit-workshop/package.json:91]: @danielamorse is this an intention change? It seems like we would like to keep the variability of the `^`?

**Review by colbytcook — APPROVED** [review]
@danielamorse I left a small comment on one of the package.json

This Pr is approved, I was able to run the following bolt command successfully.
- yarn clean
- yarn setup
- yarn 
- yarn build
- yarn start
- yarn test:js
- yarn cc n
- yarn cc y (found an unrelated error tracked in https://pegadigitalit.atlassian.net/browse/DS-899, no need to block this PR it is an existing bug)
- yarn ce
- npx jest packages/components/bolt-accordion/__tests__/accordion.js 
- npx jest packages/components/bolt-accordion/__tests__/accordion.js -u

File: packages/website-ui/uikit-workshop/package.json (hunk 1 above)
Comment (danielamorse) [reply packages/website-ui/uikit-workshop/package.json:91]: @colbytcook yeah, keeping the `^` updates us to a newer version which introduces a bug. I forget exactly what it is, something to do with Node version. Locking it down until we can just remove this library.

**Review by adamszalapski — APPROVED** [review]
All looks, good. I don't notice any issues.

**Review by remydenton — APPROVED** [review]
All checks out, thanks for this @danielamorse

**Pull request merged**